packages/pangea-sdk/pangea/request.py

```diff
@@ -76,7 +77,7 @@ def queued_support(self, value: bool):
 
         return self._queued_retry_enabled
 
-    def post(self, endpoint: str = "", data: dict = {}) -> PangeaResponse:
+    def post(self, endpoint: str = "", data: str | Dict = {}) -> PangeaResponse:
```

Comment:
@pangea-andrest it seems like CI/CD keeps failing because of this change

Error:
```
200  File "/builds/pangeacyber/pangea-python/packages/pangea-sdk/pangea/request.py", line 80, in PangeaRequest
201    def post(self, endpoint: str = "", data: str | Dict = {}) -> PangeaResponse:
202TypeError: unsupported operand type(s) for |: 'type' and '_GenericAlias'
```

Reply:
Oh, should `Dict` be `dict`?

Reply:
I think `dict` is on python 3.10 or newer, that's why we use `Dict`. I'll check it